torch/_subclasses/functional_tensor.py

```diff
@@ -249,15 +256,35 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             )
             return NotImplemented
 
+        def _can_decompose(func):
+            # TODO (tmanlaibaatar)
+            # Eventually, we don't want to decompose any aten op at all
+            # except for aten ops that lie about their schemas (ops
+            # claimed to be functional but actually aren't).
+            # Since we never tested functionalization for every CompositeImplicitAutograd
+            # ops having a direct functional kernel. As a result, we can't safely
+            # turn off decomp for predispatch right now. Therefore, we do best
```

Comment:
I'd actually argue that:

(1) the "safety" is what we are risking with this PR

(2) the "not every composite inplace op has a functional variant" is a coverage gap, but not really a safety risk, since we'll loudly error when we try to generate functionalization kernels for these new (composite) inplace/view ops.